cli/src/commands/merkle-tree/initialize.ts

```diff
@@ -36,9 +41,15 @@ class InitializeCommand extends Command {
 
   async run() {
```

Comment:
Do we have a test for this command?

Reply:
I don't think so. Probably worth adding in a separate PR.